microsoft/testsuites/core/storage.py

```diff
@@ -677,9 +696,17 @@ def _get_mtab_mount_point_regex(self, mount_point: str) -> Pattern[str]:
         return regex
 
     def _check_root_partition_in_log(
-        self, log: str, os_partition_info: PartitionInfo
+        self, log: str, os_partition_info: PartitionInfo, is_cvm: bool = False
     ) -> bool:
-        if (
+        if is_cvm:
+            if not get_matched_str(
```

Comment:
Add code comments to explain the newly added logic.

Reply:
added